pkgs/applications/office/planner/default.nix

```diff
@@ -8,9 +8,10 @@
 , gnome2
 , libxslt
 , python
+, python27Packages
```

Comment:
Never mix `python` and `pythonPackages` – they can point to completely different versions. Either use `python.pkgs` or `pythonPackages`. Also it is a good idea to use `python2` or `python3` to be specific.